### DISCUSSION TITLE
DataLinq - Diagramme und Drucken über den Browser

### DISCUSSION THREAD

**Discussion by fanie-mak** [e-netze/webgis-community#31] [Q&A]
Hallo,
kennt jemand von euch auch dieses Problem:

Wenn ich in DataLinq einen Report mit einem Diagramm erstelle und diesen dann über den Browser drucken möchte, werden dem Druck in **Chrome** irgendwelche Seiten angehängt, die jedoch leer sind (bei mir waren es zum Beispiel 511 Seiten). 
In Edge und Firefox kommt das nicht vor. 

Anbei ein Beispiellink:
https://gis.ktn.gv.at/webgisapi/datalinq/report/xx-test@xx-query@xx-view?id=20405

**Comment by gis-eni** [comment]
Ist bei mir genau so. Hat vielleicht irgendwas mit der ``chart.js`` Library zu tun.
DataLinq ist nicht wirklich zum Ausdrucken gemacht. Wenn die Seite komplizierter ist, schaut es bei mir auch in anderen Browsern komisch aus. Allerdings scheint nur Chrome das Problem mit den vielen leeren Seiten zu haben.

In der Quelle schaut alles nach normalem HTML aus. Ich weiß nicht genau, was man machen könnte. Eine neuer Version von ``chart.js`` könnte man machen, allerdings muss man dann wahrscheinlich alle DataLinq Seiten umbauen ;(
